Tools/Replay/DataFlashFileReader.cpp

```diff
@@ -7,6 +7,10 @@
 #include <unistd.h>
 #include <time.h>
 
+#ifndef PRIu64
```

Comment:
You need to `#include <cinttypes>`, otherwise it will never be defined.

Reply:
Thanks for that... I did not notice it because all CI tests passed.